pkg/install/install.go

```diff
@@ -142,11 +142,11 @@ func NewInstaller(ctx context.Context, log *logrus.Entry, env env.Interface, db
 func (i *Installer) Install(ctx context.Context, installConfig *installconfig.InstallConfig, platformCreds *installconfig.PlatformCreds, image *releaseimage.Image) error {
 	steps := map[api.InstallPhase][]interface{}{
 		api.InstallPhaseBootstrap: {
+			action(i.createBillingRecord),
 			action(i.createDNS),
 			action(func(ctx context.Context) error {
 				return i.installStorage(ctx, installConfig, platformCreds, image)
 			}),
-			action(i.createBillingRecord),
```

Comment:
Can't do this.  Billing record creation relies on knowing we own the cluster resource group, and that is only after installStorage complete.  Please fix the delete side to be tolerant of a missing record (this is needed for clusters that currently exist as well).

Reply:
Thanks, I was missing why we did this way :) will do